relayer/chain/ethereum/listener.go

```diff
@@ -28,16 +28,19 @@ type Listener struct {
 	basicOutboundChannel        *outbound.BasicOutboundChannel
 	incentivizedOutboundChannel *outbound.IncentivizedOutboundChannel
 	mapping                     map[common.Address]string
-	address                     common.Address
```

Comment:
can delete this file right?

Reply:
Yeah. I'll do it in a follow-up

Reply:
Deleting it in https://github.com/Snowfork/polkadot-ethereum/pull/367